### DISCUSSION TITLE
Regarding the issue of backpack firmware upgrade with the internal module ESP32-S3

### DISCUSSION THREAD

**Discussion by norulers** [ExpressLRS/ExpressLRS#2647] [General]
Using the ESP32-S3 as an internal module, with GPIO 44 and 43 connected to the remote control, GPIO 14 and 13 connected to the backpack serial port, GPIO 21 connected to boot, and GPIO 38 connected to EN, is it possible to use EDGETX passthrough to upgrade the backpack firmware?

**Comment by pkendall64** [comment]
It should be yes. The work was do to support passthrough flashing, and ETX passthrough.
